.devcontainer/default/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM swiftlang/swift:nightly-main-jammy
+
+RUN \
+  #
+  # Disable apt interactive prompts for this RUN command
+  export DEBIAN_FRONTEND="noninteractive" && \
+    # 
+    # Update apt package list
+    apt-get update && \
+    # 
+    # Install sourcekit-lsp dependencies
+    apt-get install -y \
+      libsqlite3-dev libncurses5-dev python3 ninja-build && \
```

Comment:
It’s been a while since the PR was opened and `ninja-build` is no longer a dependency. So I think we can remove that.